docs/tutorials/structural_tag.md

```diff
@@ -61,7 +61,19 @@ The format field requires a format object. We provide several basic format objec
     }
     ```
 
-3. `sequence`
+3. `grammar`
+
+    This format can be used to match a given ebnf grammar.
+    ```json
+    {
+        "type": "grammar",
+        "grammar": "..."
+    }
+    ```
+
+    We can use it as the context of other structural tags as well.
```

Comment:
Add a sentence:

When using grammar constraints, you need to be especially careful. If the grammar is too general (for example `.*`), it will cause the subsequent constraints to become ineffective.